client/src/utils/fight/hypnotise.ts

```diff
@@ -1,6 +1,8 @@
 /* eslint-disable no-void */
 import { FIGHTER_HEIGHT, FIGHTER_WIDTH, HypnotiseStep } from '@labrute/core';
 import { sound } from '@pixi/sound';
+// eslint-disable-next-line import/no-extraneous-dependencies
```

Comment:
No need for this eslint rule to be disabled, filter-glow is already part of the dependencies